### PR TITLE
Icon param for radio choice options.

### PR DESCRIPTION
**Pull request by k0nG**
TP:
https://rockabox.tpondemand.com/entity/9537

Allows icons to appear next to radio choices.

### CODE REVIEW

**Comment by ohskylab** [comment]
@k0nG Mind updating the changelog as part of this PR?

**Comment by k0nG** [comment]
I do not mind at all young sir. On it now.

**Comment by k0nG** [comment]
Updated changelog.

**Comment by ohskylab** [comment]
Thank you kindly.

File: src/rb-radio-control/demo/demo.tpl.html [hunk 1]
```diff
@@ -70,6 +70,24 @@
...
+            Icon modifier
+        </h4>
+    </div>
+    <div class="ComponentView">
+        <form name="iconRadioForm">
+            <rb-radio-control
+                choices="demoCtrl.iconChoices"
+                ng-model="demoCtrl.selectedIcon"
+                form="iconRadioForm"
+                name="ng-radio--row-group">
```
Comment (ohskylab) [inline src/rb-radio-control/demo/demo.tpl.html]: @k0nG Could you update `name` in the demo so the group has it's own labels?

Comment (ohskylab) [inline src/rb-radio-control/demo/demo.tpl.html]: :ok_hand:

**Pull request merged**